docs/RELEASE.md

```diff
@@ -0,0 +1,62 @@
+# Processes
+
+The following processes are necessary to actualize our releases. Each process has a *Cadence* on which it must execute and an *Responsible* that is responsible for autonomously doing so and reporting back any error in the RelEng<sup>1</sup> channel.
+
+## Mainline Release
+
+Cadence: every two weeks. Responsible: Release Team.
+
+This process aims to release the `stable` branch as a *Mainline* release every two weeks. It needs to start three days before the proposed release deadline to allow leeway for unexpected issues. This process should eventually be automated.
```

Comment:
If this should be automated at some point, I would not add any extra "leeway". If something unexpected happens, I think it is fine that it takes longer, but "unexpected" should not be the norm. Otherwise, we need to check the process and improve it.

Reply:
Okay I will remove it.